UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/FunctionalTests/ExchangeSetForLargeFile.cs

```diff
@@ -157,12 +164,29 @@ public async Task WhenICallExchangeSetApiWithMultipleProductIdentifiers_ThenEncF
                     }
                     baseNumber++;
                     var folderName = $"B{baseNumber}";
-                    checkFolder = FssBatchHelper.CheckforFolderExist(DownloadedFolderPath, folderName);
+                    checkFolder = FssBatchHelper.CheckforFolderExist(folderPath, folderName);
                 } while (checkFolder);
+
+                mediaNumber++;
+            }
+        }
+
+        [Test]
+        [Category("SmokeTest")]
+        public void WhenICallExchangeSetApiWithAnInValidProductVersion_ThenAProductTxtFileIsGenerated()
+        {
+            int mediaNumber = 1;
```

Comment:
I don't think mediaNumber is used in this function, other than to increment. Could you confirm and remove?

Reply:
Updated.

Thanks